protos/quiz.proto

```diff
@@ -1,4 +1,35 @@
 syntax = "proto3";
 
-message ClassQuizQuestion {
+import "protos/status.proto";
+
```

Comment:
a jakby tu zrobic osobny serwis od dolaczania do quizu? mznaby wywalic quiz z serwisu zajec

bo jak uczen dolaczy to bedzie mial zapisanye uuid zajec